package.json

```diff
@@ -28,7 +28,30 @@
         "start": "npm-run-all build:sass --parallel watch:*",
         "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
         "lint": "eslint --ext=.js,.jsx .",
-        "fix": "npm run lint -- --fix"
+        "fix": "npm run lint -- --fix",
+        "lint:scss": "stylelint '**/*.scss'",
+        "lint:scss:fix": "stylelint '**/*.scss' --fix",
+        "prepare": "husky install",
+        "svgo": "svgo -r -f ./"
```

Comment:
Should we rename this command to be something meaningful? We have named the command for minifying images as `images`.

Reply:
yes, we can use a meaningful name.